pkg/hive/hive.go

```diff
@@ -399,9 +394,22 @@ func (h *Hive) PrintDotGraph() {
 	}
 }
 
+// SetEnvPrefix globally sets the enviroment prefix to use with the hive package.
+// The given prefix will be upper-cased and a trailing underscore is added (if not present).
+//
+// This should be used early in initialization and only once as it affects all hives
+// in the program.
+func SetEnvPrefix(prefix string) {
+	prefix = strings.ToUpper(prefix)
+	if prefix[len(prefix)-1] != '_' {
+		prefix += "_"
+	}
```

Comment:
I think this will panic if you give it an empty string, as `prefix[len(prefix)-1]` will evaluate to `prefix[-1]`.

We can use `strings.HasSuffix` to do the check safely. Although, in the case of an empty string we should probably maintain the empty string anyway, instead of setting the prefix to be "_".
```suggestion
	if prefix != "" && !strings.HasSuffix(prefix, "_") {
		prefix += "_"
	}
```

Reply:
:+1: